Add WiiM docs. (#616)
music-assistant/music-assistant.io: src/content/docs/player-support/wiim.md

### AirPlay Functionality

Some WiiM devices support the AirPlay 1 <a href="https://en.wikipedia.org/wiki/Remote_Audio_Output_Protocol" target="_blank" rel="noopener noreferrer">RAOP</a> protocol and/or the Airplay 2 protocol and this enables very useful functionality within Music Assistant.

If the WiiM device is grouped with an AirPlay device, or if the default [output protocol](/settings/individual-player/#output-protocols) is changed to `AirPlay`, then the AirPlay protocol will be used for playback. Other WiiM players can be synced with this player (even if they themselves do not have AirPlay, as the native WiiM will be used for that connection). This means it is possible to play the same audio in perfect sync to a combination of AirPlay and WiiM speakers.

> [!NOTE]
> If, as a result of grouping, a switch is required to the AirPlay protocol then a small silence due to the stream restart is to be expected.

## Settings

In addition to the [Player Provider Settings](/settings/player-provider/) when setting up this provider the following settings are available:

- <b>Manual IP addresses for discovery.</b> Not recommended for normal use. Refer to the description in the MA UI

> In addition to the [Individual Player Settings](/settings/individual-player/) the WiiMplayers have the following settings:

Reviewer: ```suggestion
In addition to the [Individual Player Settings](/settings/individual-player/) the WiiM players have the following settings:
```

> In addition to the [Individual Player Settings](/settings/individual-player/) the HEOS players have the following settings:

Reviewer: ```suggestion
In addition to the [Individual Player Settings](/settings/individual-player/) the WiiM players have the following settings:
```
You fixed my cut and paste error on the Sonos page and then did the same here :-)

- <b>Enforce gapless playback with queue flow mode streaming.</b> Enabling this option will send all tracks as a contnuous audio stream. Use for players that dont natively support gapless or crossfading. Can also help with players that have difficulty transitioning between tracks. May have the side effect of losing metadata to the player

## Known Issues / Notes

- No sound during grouped playback: When multiple WiiM devices are part of an Airplay group, and one of those WiiM devices does not support Airplay, this could result in silence. For example, grouping a Sonos with a WiiM amp (Airplay) and a WiiM sound (no Airplay). Changing the Airplay protocol of the 'bridge device' (i.e. the device that does support Airplay) to 'Prefer Airplay 2' in the advanced settings fixes this
- Song restarted when grouping a WiiM device: A song is restarted the first time a WiiM device is grouped to another WiiM device. This looks like something in the WiiM firmware itself where it is (probably) changing to a 'group playback mode' ensuring compatible sample rate / bit depth. Any subsequent group/ungroup actions should not be affected. We are looking for a workaround for this

> - Album art sometimes not visible in the WiiM app: Album arts are sometimes not shown in the WiiM app, because the WiiM firmware cannot handle long URLs sent out by our imageproxy. We are looking into a way to fix this.

Reviewer: ```suggestion
- Album art sometimes not visible in the WiiM app: Album art is sometimes not shown in the WiiM app because the WiiM firmware cannot handle long URLs sent out by our imageproxy. We are looking into a way to fix this.
```
